### PR TITLE
[Enhancement] Storage Add a common write method to avoid missing some properties when writing files

### LINKED ISSUE #6833: When use Storage#writeFrontendRoleAndNodeName to write file will only write ROLE and name, other properties which in that file will be delete
### Steps to reproduce the behavior (Required)
1. CREATE TABLE '...'
2. INSERT INTO '....'
3. SELECT  '....'

### StarRocks version (Required)
 - You can get the StarRocks version by executing SQL `select current_version()`

### PR DESCRIPTION
**Pull request by waittttting**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [x] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #6833

## Problem Summary(Required) ：

Storage Add a common write method to avoid missing some properties when writing files

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 27 / 30 (90.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/persist/Storage.java|27|30|90.00%|

File: fe/fe-core/src/main/java/com/starrocks/persist/Storage.java [hunk 1]
```diff
@@ -53,22 +55,21 @@ public class Storage {
...
+    private Map<String, String> roleFileMap = new HashMap<>();
+    private Map<String, String> versionFileMap = new HashMap<>();
+    
     private long imageJournalId;
     private String metaDir;
 
     public Storage(int clusterID, String token, String metaDir) {
-        this.clusterID = clusterID;
-        this.token = token;
+        versionFileMap.put(CLUSTER_ID, clusterID + "");
```
Comment (padmejin) [inline fe/fe-core/src/main/java/com/starrocks/persist/Storage.java:65]: why add an empty string?

File: fe/fe-core/src/main/java/com/starrocks/persist/Storage.java [hunk 2]
```diff
@@ -86,9 +87,9 @@ public void reload() throws IOException {
         if (versionFile.isFile()) {
             try (FileInputStream in = new FileInputStream(versionFile)) {
                 prop.load(in);
-                clusterID = Integer.parseInt(prop.getProperty(CLUSTER_ID));
+                versionFileMap.put(CLUSTER_ID, prop.getProperty(CLUSTER_ID));
```
Comment (padmejin) [inline fe/fe-core/src/main/java/com/starrocks/persist/Storage.java:90]: Since we already use java properties to load/store, why not use it in memory?

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 31 / 34 (91.18%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/persist/Storage.java|31|34|91.18%|

File: fe/fe-core/src/main/java/com/starrocks/persist/Storage.java (hunk 2 above)
Comment (waittttting) [reply fe/fe-core/src/main/java/com/starrocks/persist/Storage.java:91]: I think using map to store kv is more in line with everyone's programming habits. java property is just used to write files

File: fe/fe-core/src/main/java/com/starrocks/persist/Storage.java [hunk 3]
```diff
@@ -48,27 +50,27 @@ public class Storage {
...
     public static final String IMAGE = "image";
     public static final String IMAGE_NEW = "image.ckpt";
     public static final String VERSION_FILE = "VERSION";
     public static final String ROLE_FILE = "ROLE";
 
-    private int clusterID = 0;
-    private String token;
-    private FrontendNodeType role = FrontendNodeType.UNKNOWN;
-    private String nodeName;
+    private Map<String, String> roleFileMap = new HashMap<>();
```
Comment (gengjun-git) [inline fe/fe-core/src/main/java/com/starrocks/persist/Storage.java:59]: Better not use map to store the deserialized property, the deserialized property should be stored to a definitive var, like toke, nodeName.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 19 / 19 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/persist/Storage.java|19|19|100.00%|

**Review by padmejin — APPROVED** [review]
LGTM

**Review by gengjun-git — APPROVED** [review]

**Review by Astralidea — APPROVED** [review]

**Pull request merged**